gsy_e_sdk/redis_device.py

```diff
@@ -1,21 +1,17 @@
-from gsy_e_sdk.constants import LOCAL_REDIS_URL
-from gsy_e_sdk.redis_client_base import RedisClientBase
+import logging
+from gsy_e_sdk.clients.redis_asset_client import RedisAssetClient
 
+logger = logging.getLogger(__name__)
+console_handler = logging.StreamHandler()
+console_handler.setFormatter(logging.Formatter("%(levelname)s: %(message)s"))
+logger.addHandler(console_handler)
```

Comment:
shouldn't the cli module already handle these settings?

Reply:
This handler works when the cli is not used. But you're right in the sense that if the cli is used two warnings are shown (one for each handler). If you want, I can remove this in a new PR.